serial.py

```diff
@@ -22,12 +23,17 @@ class SerialDisconnectException(SerialException):
 
 class Serial:
 
-    BAUD_MAP = { 9600: termios.B9600, 115200: termios.B115200 }
+    BAUD_MAP = {
+        9600: termios.B9600,
+        19200: 14,
```

Comment:
Can you add comment what is 14/where it comes from?

Reply:
termios baud rate constants have integer values which ultimately refer to positions in the tty baud rate table in the [linux kernel](https://github.com/torvalds/linux/blob/894025f24bd028942da3e602b87d9f7223109b14/drivers/tty/tty_baudrate.c).  Micropython's termios does not define all of the baud rate constants, but they can be referred to by their integer values.

Reply:
Thanks for the details, I actually added it as an inline comment too.